### DISCUSSION TITLE
re-consider downgrading to ffmpeg 6

### DISCUSSION THREAD

**Discussion by ivanjx** [blakeblackshear/frigate#13822] [General]
on my system (i7 7700T) it adds almost 2x of cpu % for my 2K camera compared to ffmpeg 7.
command:
```
"/usr/lib/ffmpeg/7.0/bin/ffmpeg -hide_banner -loglevel warning -threads 2 -hwaccel qsv -qsv_device /dev/dri/renderD128 -hwaccel_output_format qsv -c:v h264_qsv -user_agent FFmpeg Frigate/0.15.0-b729b29 -rtsp_transport tcp -timeout 5000000 -i rtsp://127.0.0.1:8554/cam1 -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy /tmp/cache/cam1@%Y%m%d%H%M%S%z.mp4 -r 5 -vf vpp_qsv=framerate=5:w=2688:h=1520:format=nv12,hwdownload,format=nv12,format=yuv420p -threads 2 -f rawvideo -pix_fmt yuv420p pipe:"
```

**Comment by NickM-27** [comment]
It was not stable for me during extensive testing, jellyfin also is not using ffmpeg 7 yet. The odds are very high that ffmpeg 7 will not be stable for the majority of users which is crucial. The path is configurable, there's nothing stopping you from using ffmpeg 7 if you want to

**Reply by NickM-27** [reply]
We were able to find a way to use the latest intel driver despite being on an older base image, users have tested and confirmed that this has fixed their issues with qsv so ffmpeg 7 should be a viable option again
